crates/runtime/src/accelerated_table/refresh_task_runner.rs

```diff
@@ -107,27 +107,16 @@ impl RefreshTaskRunner {
                                 }
                             }
                         },
-
```

Comment:
Not explicitly related to this PR, but avoids having the additional `if let Some(overrides_opt) = overrides_msg`.